gulp/util/bundleLogger.js

```diff
@@ -8,19 +8,21 @@
 import gutil        from 'gulp-util';
 import prettyHrtime from 'pretty-hrtime';
 
-var startTime;
-
 export default {
 
   start() {
-    startTime = process.hrtime();
-    gutil.log('Running', gutil.colors.green('\'bundle\'') + '...');
+    let startTime = process.hrtime();
+    gutil.log('Starting', `
+      '${gutil.colors.cyan('bundle')}'...
```

Comment:
could you put these `gutil.log` params on the same line? They don't look long enough to need wrapping